### ISSUE TITLE
Accessibility issue - selected date is not announced by screen reader

### ISSUE THREAD

**Issue by Ke1sy** [Hacker0x01/react-datepicker#3559]
**Describe the bug**
When you select a date with keyboard, it is expected that the new date will be announced by screen reader, but nothing is announced.

**To Reproduce**
1. Start the screen reader and open the date-picker with keyboard.
2. Navigate to any date and click on it.

**Expected behavior**
Selected date will be announced by screen reader.

**Desktop (please complete the following information):**
- OS: Windows 10
- Browser: Chrome (v. 100.0.4896.75), Firefox

**Comment by martijnrusschen** [comment]
Fixed by https://github.com/Hacker0x01/react-datepicker/pull/3817
